app/package.json

```diff
@@ -64,7 +64,7 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "git-revision-webpack-plugin": "^5.0.0",
     "html-webpack-plugin": "^5.3.2",
-    "mini-css-extract-plugin": "^1.6.1",
+    "mini-css-extract-plugin": "^2.1.0",
```

Comment:
How can the bot upgrade caret 2 to 3 as major version?

Reply:
Don't worry. This is from another PR that I merged a sec ago. It takes a little bit but the bot is resolving and rebasing automatically.